FlowCrypt/Controllers/Setup/SetupInitialViewController.swift

```diff
@@ -152,14 +153,18 @@ extension SetupInitialViewController {
     }
 
     private func checkEKMKeys() {
-        organisationalRulesService.getEmailKeyManagerPrivateKeys()
+        emailKeyManagerApi.getPrivateKeys()
             .then { [weak self] result in
-                if result.keys.isNotEmpty {
-                    self?.showToast(
-                        "organisational_rules_ekm_private_keys_message".localizeWithArguments(result.keys.count, result.urlString ?? "")
-                    )
-                }
+                let urlString = self?.emailKeyManagerApi.getPrivateKeysUrlString() ?? ""
```

Comment:
At this point since we pulled from EKM using this URL, the URL must exist. Missing URL is completely unexpected, ever, and therefore should be a fatal error, not an empty string.

Reply:
hm, in scope of ClientConfiguration you said that any field of it may be null and `keyManagerUrl` is the field of ClientConfiguration. What should we do in this case?